src/main/java/org/openelisglobal/common/rest/provider/PatientSearchRestController.java

```diff
@@ -86,6 +106,12 @@ public PatientSearchResultsForm getPatientResults(HttpServletRequest request,
                     return form;
                 }
             }
+
+            if (request.getParameter("crResult") != null && request.getParameter("crResult").contains("true")) {
+                List<PatientSearchResults> fhirResults = searchPatientInClientRegistry(nationalID);
```

Comment:
Here you will serach with lastName ,firstName ,nationalID,subjectNumber ,dateOfBirth and gender from the ClentRegistry

Reply:
Yeah sure!

Reply:
will change this in the next commit